<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HmiSaveDialog</class>
 <widget class="QDialog" name="HmiSaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>HmiSaveDialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_txt">
      <attribute name="title">
       <string>txt檔案</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="3">
        <widget class="QTextEdit" name="textEdit_txt"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_txtSave">
         <property name="text">
          <string>儲存檔案</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_txtClose">
         <property name="text">
          <string>關閉</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>220</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_mat">
      <attribute name="title">
       <string>mat檔案</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_matSave">
         <property name="text">
          <string>儲存檔案</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_matClose">
         <property name="text">
          <string>關閉</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="4">
        <widget class="QTableWidget" name="tableWidget_mat">
         <column>
          <property name="text">
           <string>變數名稱</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>資料型態</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>數量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>總資料大小(bytes)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableWidget_mat</tabstop>
  <tabstop>pushButton_matSave</tabstop>
  <tabstop>pushButton_matClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
